homeassistant/components/here_weather/const.py

```diff
@@ -0,0 +1,397 @@
+"""Constants for the HERE Destination Weather service."""
+DOMAIN = "here_weather"
+
+HERE_API_KEYS = "here_api_keys"
+
+DEFAULT_SCAN_INTERVAL = 120
+
+CONF_LANGUAGE = "language"
+CONF_OFFSET = "offset"
+CONF_OPTION = "option"
+
+MODE_ASTRONOMY = "forecast_astronomy"
+MODE_HOURLY = "forecast_hourly"
+MODE_DAILY = "forecast_7days"
+MODE_DAILY_SIMPLE = "forecast_7days_simple"
+MODE_OBSERVATION = "observation"
+CONF_MODES = [
+    MODE_ASTRONOMY,
+    MODE_HOURLY,
+    MODE_DAILY,
+    MODE_DAILY_SIMPLE,
+    MODE_OBSERVATION,
+]
+DEFAULT_MODE = MODE_DAILY_SIMPLE
+
+ASTRONOMY_ATTRIBUTES = {
+    "sunrise": {"name": "Sunrise", "unit_of_measurement": None},
```

Comment:
Add a new option to influence the `entity_registry_enabled_default` entity property. Many entities should be disabled by default.

Reply:
Can you give a guideline on which sensors should be enabled and which not?

Reply:
Disable all sensors. Keep the weather enabled by default.

Reply:
Done